code/modules/research/designs/tool_designs.dm

```diff
@@ -63,6 +63,16 @@
 	category = list("Tool Designs")
 	departmental_flags = DEPARTMENTAL_FLAG_ENGINEERING
 
+/datum/design/rld
+	name = "Rapid Ligth Dispencer"
```

Comment:
"Light" is misspelled here and in the description

Reply:
fixing em typos